content/doc/book/pipeline/syntax.adoc

```diff
@@ -1257,9 +1257,10 @@ pipeline {
 Stages in Declarative Pipeline may declare a list of nested stages to be run
 within them in sequential order. Note that a stage must have one and only one of
 `steps`, `parallel`, or `stages`, the last for sequential stages. The stages
-within `stages` in a stage cannot contain further `parallel` or `stages`
-themselves, but they do allow use of all other functionality of a stage, including
-`agent`, `tools, `when`, etc.
+within `stages` in a stage cannot contain further `stages` themselves,
+and can only contain `parallel` if the parent `stage` is not already
+within a higher-level `parallel`. The parent stages do allow use of
+all other functionality of a stage, including `agent`, `tools, `when`, etc.
```

Comment:
Thanks for requesting my review @abayer ... I think the overuse of the word 'stage' makes these last couple of sentences a tad difficult to follow ... Therefore, please change ...

> The stages within `stages` in a stage cannot contain further `stages` themselves, and can only contain `parallel` if the parent `stage` is not already within a higher-level `parallel`. The parent stages do allow use of all other functionality of a stage, including `agent`, `tools, `when`, etc.

... to (if this is still correct) ...

> It is not possible to nest a `parallel` block within a `stage` directive if that `stage` directive is nested within a `parallel` block itself. However, a `stage` directive within a `parallel` block can use all other functionality of a stage, including `agent`, `tools, `when`, etc.

If I'm missing something here, please mention it here.

Reply:
Nice, that's definitely better. =)